conjure-java-core/src/main/java/com/palantir/conjure/java/services/ServiceGenerator.java

```diff
@@ -49,17 +49,24 @@
         return emittedPaths;
     }
 
-    static Optional<String> getJavaDoc(EndpointDefinition endpointDef) {
+    static Optional<String> getJavaDoc(EndpointDefinition endpointDef, boolean includeRequestLine) {
```

Comment:
nit: boolean args aren't the easiest to read at call sites, thoughts on making this private and exposing `getJavaDoc` and `getJavaDocWithRequestLine`?